packages/app/cypress.config.ts

```diff
@@ -10,6 +10,7 @@ export default defineConfig({
     runMode: 2,
     openMode: 0,
   },
+  videoCompression: false, // turn off video compression for CI
```

Comment:
Is there a reason this isn't the default in Cypress?

Reply:
historical reasons and people not recording to the dashboard